<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenAlgorithmClass</class>
 <widget class="QMainWindow" name="GenAlgorithmClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GenAlgorithm</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEditAccuracy">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRun">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditPopulationSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>10</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditGenerationCount">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditStrategyPercent">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>3</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditInversionProbability">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSelectionMethods">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>120</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Selection methods:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMS1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Best ones</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMS2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Roulette wheel</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMS3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tournament</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditSelectionParam">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>6</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSelectionInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Saved creatures [%]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecombination">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>120</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Crossover:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonRecomb1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>One point</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRecomb2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Two points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRecomb3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Three points</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRecomb4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Homogenous</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecombination_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>120</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Mutation:</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMutation1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Boundary</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMutation2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>One point</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMutation3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Two points</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelAccuracy">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPopSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Population size</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditCrossoverProbability">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>6</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditMutationProbability">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIterCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Iteration count</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIterCount_">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Inversion probability [%]</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelElitism">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Elitism - how many best &lt;/p&gt;&lt;p&gt;results have to survive&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRunChart">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run chart</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExecutionTime">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Execution time (milliseconds):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GenAlgorithm.qrc"/>
 </resources>
 <connections/>
</ui>
